Codeinwp/neve
# enh: header search component background will inherit site background

### Summary
Makes the Search icon component search forms inherit the website background;
The Close button `×` should now inherit the website text color; 


### Will affect visual aspect of the product

YES - but I guess they're not going to be visible inside the visual regression

### Screenshots 
![image](https://user-images.githubusercontent.com/15010186/106613796-4056b200-6573-11eb-96d4-de14ae583926.png)
![image](https://user-images.githubusercontent.com/15010186/106613834-506e9180-6573-11eb-87d6-e2a4b2332225.png)
![image](https://user-images.githubusercontent.com/15010186/106613875-5bc1bd00-6573-11eb-844a-6b5ab5c224c0.png)


### Test instructions

- Add the Search Icon to a header row;
- The search icon component form should inherit the website background in all `Canvas`, `Minimal`, and `Float about header` open behaviors;

cc @mghenciu do you feel this is a good enough solution? :D 


Closes #2376.
